gradle/libs.versions.toml

```diff
@@ -1,5 +1,5 @@
 [versions]
-androidGradlePlugin = "8.3.2"
```

Comment:
why?

Reply:
I was getting this error:
The project is using an incompatible version (AGP 8.3.2) of the Android Gradle plugin. Latest supported version is AGP 8.2.2
See Android Studio & AGP compatibility options.